docs/pwa.md

```diff
@@ -0,0 +1,51 @@
+---
+description: >-
+  Boltz can be installed as Progressive Web App (PWA) on Android and iOS devices
+  as well as desktop computers.
+---
+
+# 📲 Install as App
+
+## Android
+
+1. Open [boltz.exchange](https://boltz.exchange) in a mobile browser like Chrome/Chromium, open the browser menu and tap "Install app":
```

Comment:
```suggestion
1. Open [boltz.exchange](https://boltz.exchange) in a mobile browser like Chrome or Chromium, open the browser menu and tap "Install app":
```

Is there a Chromium version on Android?

Reply:
No official one, you are right. Replaced with "Vanadium" which is the standard "hardened Chromium" browser on GrapheneOS which I suspect a lot of our users will be using.